how-to/rotate/gpg_signing_key.rst

```diff
@@ -69,9 +80,9 @@ The new signing subkey will be deployed to the production instance of `Autograph
 
     e) Add new.key to signingscript and point the `firefoxci-adhoc-3` pool's `GPG_PUBKEY_PATH` variable at it.
```

Comment:
Should this section also mention updating the corresponding keyid in signingscript/docker.d/passwords.yml before opening a PR with the changes?  Otherwise, if we're not rotating credentials either, we'll keep signing with the old subkey.

Reply:
If that's the way we go, yeah, it should. In the past I think this has been accomplished by relying on default keyids, so we only needed to change credentials.

If you don't mind, could you pick that up? (I'm assuming there's going to be other enhancements/changes to this needed if we start rotating that way.)